app/experimenter/experiments/tasks.py

```diff
@@ -198,6 +198,11 @@ def update_status(experiment):
         if experiment.status == Experiment.STATUS_COMPLETE:
             bugzilla.update_bug_resolution(experiment)
 
+    if recipe_data and (is_paused(recipe_data) != experiment.is_paused):
+        with transaction.atomic():
```

Comment:
Not exactly sure why this is needed, I'm assuming it has to do with the transaction.atomic() above, but without it I get a django.db.transaction.TransactionManagementError from time to time

Reply:
Huh interesting, it should complete the transaction when the with block ends...

Reply:
Well at any rate in theory we should be using that decorator more so I guess this is good?